include/linux/hax_interface_linux.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2011 Intel Corporation
```

Comment:
Sorry, I didn't pay much attention to the file headers, so the authorship is certainly inconsistent. 

Will do the changes you suggested! I've never seen the *Original work* or *Modified work* prefixes either. I think we should drop these prefixes and simply list author copyright notices sorted by date (descending as time increases?). E.g.:
```
/*
 * Copyright (c) 2011 Intel Corporation
 * Copyright (c) 2018 Kryptos Logic
 * [...]
```
So if a third party forks and modifies this file, he can simply add his name below.
(I will apply these changes to all files that contain code originally authored by Intel).

Reply:
> descending as time of first modification increases?

The example looks good to me, thanks! (I think it uses the _ascending_ order of year.)

> So if a third party forks and modifies this file, he can simply add his name below.

Exactly. Also I think it's a common practice to aggregate the notices by author. So if Intel updates the file substantially next year, we'll just update the first record (`2011, 2019`) instead of adding a new one.